ruby-challenges.rb

```diff
@@ -22,13 +36,30 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+p beatles_album1.delete "aeiou"
+p beatles_album2.delete "e"
+p beatles_album3.delete "A,a,e,i,o,u"
 
 
-# -------------------3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forward or backward. Use the test variables provided.
+# # -------------------3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forward or backward. Use the test variables provided.
 
 palindrome_test_case1 = 'Racecar'
 # Expected output: 'Racecar is a palindrome'
 palindrome_test_case2 = 'LEARN'
 # Expected output: 'LEARN is not a palindrome'
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def check_palindrome(string)
+    if string.reverse == string 
+      puts "#{ string } is a palindrome."
+    else 
+      puts "#{ string } is not a palindrome."
+    end
+  end
+
+  p check_palindrome(palindrome_test_case1.downcase)
+  p check_palindrome(palindrome_test_case2.downcase)
+  p check_palindrome(palindrome_test_case3.downcase)
+
+#   I seen that how I wrote the method knew it was written properly, but I kept getting 'Racecar' and 'Rotator' back as not a plindrome. I assumed it was case sensitive so I decided to add downcase so that the data matched exactly forward and backwards. Same problem with the nil at the bottom of each answer.
```

Comment:
It would be better to add the downcase method in your method itself, not in the argument. As for the nil, same as I wrote above, it's because you have puts in your method and then p with your method calls.